src/utils/api.js

```diff
@@ -1,7 +1,22 @@
 /* @flow */
+import { getToken } from './tokenStorage';
+
+export type LoginType = {
+  email: string,
+  password: string,
+};
+
+export type PromiseFormType = {
+  name: string,
+  shortDescription: string,
+  longDescription: string,
+  parliamentId: number,
+};
 
 export const checkStatus = (response: Response) => {
-  if (response.status >= 200 && response.status < 300) { //eslint-disable-line
+  /* eslint-disable no-magic-numbers */
+  if (response.status >= 200 && response.status < 300) {
```

Comment:
I suppose https://github.com/prettymuchbryce/node-http-status covers this.

I think this particular case should have the numbers though, to really drive home the point that it's checking the entire 2XX success range.

I would get more confused if I read
```
if (response.status >= HttpStatus.OK && response.status < HttpStatus.MULTIPLE_CHOICES) {
...
```